src/openforms/authentication/types.py

```diff
@@ -80,3 +80,15 @@ class EHMandate(TypedDict):
 class EHerkenningMachtigenContext(EHerkenningContext):
     representee: EHRepresenteeEntity
     mandate: EHMandate
+
+
+class EmployeeRepresenteeEntity(TypedDict):
+    identifierType: Literal["employee_id"]
```

Comment:
I'd use either `opaque` or `custom` here. I gravitate towards the former because it's already used for the eHerkenning acting subject too

Reply:
Check, yeah i'll use `opaque` then it that case, to keep things consistent 👍